code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -180,6 +180,10 @@
 		var/i = 0 //Tally up our victims.
 
 		for(var/mob/living/carbon/human/victim in range(radius,src)) //Loop through all nearby victims, including the tile.
+
+			if(istype(victim.wear_suit,/obj/item/clothing/suit/storage/marine/specialist/melee))
```

Comment:
Let's make an acid armor type instead of snowflake checks.
These are very unmaintainable.